Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -233,17 +234,22 @@ bool CruRawReader::processHBFs(int datasizealreadyread, bool verbose)
           break;
         case 1:
           LOG(info) << "all good parsing half cru";
+          LOG(info) << " mHBFoffset32:" << mHBFoffset32 << " and mTotalHBFPayload/4 : " << mTotalHBFPayLoad / 4;
           break;
         case 2:
           LOG(info) << "all good parsing half cru was blank double 0xe event";
           break;
-        default:
-          return true;
+          //        default:
+          //          return true;
       }
     }
     //take care of the case where there is an "empty" rdh containing all 0xeeeeeeee as payload.
-    if (mTotalHBFPayLoad / 4 - mHBFoffset32 == 8 && mHBFPayload[mHBFoffset32 + 7] == o2::trd::constants::CRUPADDING32) {
-      mHBFoffset32 += 8;
+    //    if (mTotalHBFPayLoad / 4 - mHBFoffset32 == 8 && mHBFPayload[mHBFoffset32 + 7] == o2::trd::constants::CRUPADDING32) {
+    //      mHBFoffset32 += 8;
+    //    }
+    if (halfcruprocess == -2) {
```

Comment:
Why is this not in the switch/case statement above? It seems like the case 2, but with a minus sign.

Reply:
The stuff above that is actually on for verbose style outputs.

Reply:
-2 is basically the case of dump and run. I would put in an enum and a pretty name, but this is hopefully that last time I look at this code ;-)

Reply:
Ok.